src/libraries/Microsoft.Extensions.Caching.Memory/src/CacheEntry.cs

```diff
@@ -84,7 +84,11 @@ internal long AbsoluteExpirationTicks
                 {
                     DateTimeOffset expiration = value.GetValueOrDefault();
                     _absoluteExpirationTicks = expiration.UtcTicks;
+#if NET8_0_OR_GREATER
```

Comment:
nit: 8.0 is the minimum version these days.
https://github.com/dotnet/runtime/blob/4b02c51f7158ca9abc68b331ac5ad64d716a0e2a/Directory.Build.props#L102

therefore:
```suggestion
#if NET
```
is enough.

Reply:
Both work in practice, but the explicit version number seems more correct.

Reply:
We are moving to version agnostic variant where we can. This code does not need the version.